webapp/portlet/src/main/webapp/external-spaces-list/externalSpacesList.js

```diff
@@ -0,0 +1,50 @@
+import { spacesConstants } from '../js/spacesConstants.js';
+
+
+export function getExternalSpacesList(offset, limit) {
+  return fetch(`${spacesConstants.SOCIAL_USER_API}${eXo.env.portal.userName}/spaces?offset=${offset || 0}&limit=${limit|| 10}&returnSize=true`, {
+    method: 'GET',
+    credentials: 'include',
+  }).then((resp) => {
+    if(resp && resp.ok) {
+      return resp.json();
+    }
+    else {
+      throw new Error ('Error when getting spaces of current user');
+    }
+  });
+}
+
+export function getExternalSpacesRequests() {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spacesMemberships?status=invited&limit=5&returnSize=true`, {
```

Comment:
```suggestion
  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spacesMemberships?status=invited&limit=5`, {
```
offset and limit to be verified